docs/testplan/automatic-account-freeze-unfreeze.md

```diff
@@ -0,0 +1,33 @@
+|                  | Don't Limit ls                                          | While account is frozen, user should be able to list objects in bucket                                                                                                                                                                                                                                                                                         | If we show them the files they could download after fixing the payment problem they might be more willing to pay                                                                                         |
+|                  | Don't Limit Delete                                      | While account is frozen, user should be able to delete objects in bucket                                                                                                                                                                                                                                                                                       | If we allow the customer to delete all files it will reduce the amount of work we have to perform.                                                                                                       |
+|                  | Limit Create Bucket                                     | While account is frozen, user should not be able to create buckets                                                                                                                                                                                                                                                                                             |                                                                                                                                                                                                          |
+|                  | Limit Connection to Buckets                             | User should not be able to use UI or cli tools to interact with or access buckets at all while account is frozen                                                                                                                                                                                                                                               | maybe give an error message in cli when trying to interact with buckets on a frozen account                                                                                                              |
```

Comment:
* "While account is frozen, user should be able to list objects in bucket"
* "User should not be able to [...] access buckets at all while account is frozen"

These requirements seem to conflict because I interpret the action of listing the objects in a bucket as a form of bucket access. Could you expound @nadimhq?